charts/hedera-mirror-common/templates/zfs/rbac.yaml

```
@@ -0,0 +1,53 @@
+{{- if .Values.zfs.enabled -}}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels: {{ include "hedera-mirror-common.labels" . | nindent 4 }}
+  name: label-reader
+  namespace: {{ include "hedera-mirror-common.namespace" . }}
+rules:
+  - apiGroups: [""]
+    resources: ["nodes"]
+    verbs: ["get", "list", "update", "patch"]
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  labels: {{ include "hedera-mirror-common.labels" . | nindent 4 }}
+  name: label-reader
+  namespace: {{ include "hedera-mirror-common.namespace" . }}
+subjects:
+  - kind: ServiceAccount
+    name: {{ .Values.zfs.init.serviceAccount.name }}
+    namespace: {{ include "hedera-mirror-common.namespace" . }}
+roleRef:
+  kind: ClusterRole
+  name: label-reader
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  labels: {{ include "hedera-mirror-common.labels" . | nindent 4 }}
+  name: configmap-updater
```

The namespace level RBAC should include both the release name and `zfs` in the name

```suggestion
  name: {{ include "hedera-mirror-common.fullname" . }}-zfs-configmap-updater
```

---

makes sense. Thanks for pointing this out.